bucket/posh-git-nightly.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "http://dahlbyk.github.io/posh-git/",
+    "description": "A PowerShell module which provides Git/PowerShell integration.",
+    "version": "2019-12-01-2ad946347e",
```

Comment:
```suggestion
    "version": "2019.12.01-05.15.57",
```

Reply:
Inclusion of the head of the commit ID allows for easier figuring out which posh-git commit you're on for issues?

Reply:
If you use commit ID as version number, app updating may break, since the IDs aren't sorted alphabetically. [`8115fe5`](https://github.com/dahlbyk/posh-git/commit/8115fe507b3ff304d3bf04ed4063f62ba8eaa034) may be followed by [`1e84ae8`](https://github.com/dahlbyk/posh-git/commit/1e84ae8741d72dd3c640bf9496d5c81116b45355) in the same day, which is not recognized by [Excavator](https://github.com/ScoopInstaller/Excavator).